api/src/main/proto/stellarstation/api/v1/radio/radio.proto

```diff
@@ -51,11 +51,125 @@ message CommunicationProtocol {
   oneof Framing {
     // AX.25 protocol settings.
     AX25 ax25 = 1;
+
+    // CCSDS framing settings.
+    CCSDS ccsds = 2;
+
+    // Bit stream framing settings.
+    BitStream bit_stream = 3;
   }
 }
 
-// Communication of a stream of bits with no framing defined.
+// CCSDS framing https://public.ccsds.org/Pubs/131x0b3e1.pdf
+// It allows for a few common variations satellites use such
+// as setting a custom synchronization word, inverting the
+// convolutional coding polynomials, etc. The sequence of
+// error-correcting codes will be applied as they are defined
+// in the CCSDS standard.
+message CCSDS {
+  // Synchronization word parameters. Although CCSDS defines
```

Comment:
Which page in the PDF docments this? Want to understand it more to flesh out this comment. For now, I think it's something like

```
// Synchronization word parameters. If not set, synchronization will be enabled using the default CCSDS standard synchronization word.
```

Reply:
Ok, if I'm not mistaken the actual words are on in 9.3.5, so we can mention it

```
// Synchronization word parameters. If not set, synchronization will be enabled using the default CCSDS standard synchronization word as in 9.3.5 (Page 9-3) in the specitication.
```

Reply:
Can we tell if a protobuf is unset? I assume we look at the `enabled` parameter and if `False`, we consider it unset? To be explicit, I worded it as " If disabled, synchronization will be enabled.." but that kind of sounds confusing...